libc-test/build.rs

```diff
@@ -289,6 +298,24 @@ fn main() {
             "FILE_ATTRIBUTE_INTEGRITY_STREAM" |
             "ERROR_NOTHING_TO_TERMINATE" if mingw => true,
 
+            // these are not in musl
+            "SOL_NETLINK" |
+            "SOL_UDP" |
+            "SOL_AX25" |
+            "SOL_ATALK" |
+            "SOL_NETROM" |
+            "SOL_IPX" |
+            "SOL_ROSE" if musl => true,
```

Comment:
Could these constants be perhaps non-MUSL? I guess they're defined in the kernel so the MUSL definitions shouldn't change in theory?